salt/cloud/clouds/gce.py

```diff
@@ -2580,7 +2580,7 @@ def create(vm_=None, call=None):
     ssh_user, ssh_key = __get_ssh_credentials(vm_)
     vm_['ssh_host'] = __get_host(node_data, vm_)
     vm_['key_filename'] = ssh_key
-    __utils__['cloud.bootstrap'](vm_, __opts__)
+    ret = __utils__['cloud.bootstrap'](vm_, __opts__)
```

Comment:
I think this should probably also have `ret.update(node_dict)` similar to how linode has.